source-postgres/backfill.go

```diff
@@ -64,6 +64,14 @@ func (db *postgresDatabase) ScanTableChunk(ctx context.Context, info sqlcapture.
 	return events, nil
 }
 
+func quoteColumnNames(names []string) []string {
+	var out []string
+	for _, name := range names {
+		out = append(out, name)
```

Comment:
This doesn't seem to actually be quoting anything, just copying them to a new slice. But it also seems like it may be unused, so perhaps it just needs deleted.

Reply:
Good catch! Yeah, this was dead code and I've deleted it now.